src/store/__tests__/addressBookSlice.test.ts

```diff
@@ -32,6 +32,21 @@ describe('addressBookSlice', () => {
     })
   })
 
+  it('should ignore empty names in the address book', () => {
+    const state = addressBookSlice.reducer(
+      initialState,
+      upsertAddressBookEntry({
+        chainId: '1',
+        address: '0x2',
+        name: '',
+      }),
+    )
+    expect(state).toEqual({
+      '1': { '0x0': 'Alice', '0x1': 'Bob' },
+      '4': { '0x0': 'Charlie', '0x1': 'Dave' },
+    })
```

Comment:
nit: Could also check against the initialState here i.e. `expect(state).toEqual(initialState)`